projects/demo/src/modules/experimental/card/examples/7/index.html

```diff
@@ -0,0 +1,93 @@
+<div
+    tuiCard
+    tuiSurface
+    class="surface"
+    (click)="currentIndex = 0"
+>
+    <tui-svg
+        src="tuiIconGooglePay"
+        tuiSlot="top"
+    ></tui-svg>
+    <tui-scrollbar>
+        <div
+            tuiFade
```

Comment:
Scrolling doesn't work here:
<img width="265" alt="image" src="https://github.com/taiga-family/taiga-ui/assets/11832552/b6bd4dda-a3f2-4bd3-964f-16981acb4ef9">
I think it's better to remove fade and use negative margins so that cards scroll all the way across the tuiCard.

Reply:
fixed